doc/release-notes/5.4-release-notes.md

```diff
@@ -274,6 +274,10 @@ Add the following JVM options to the `<config name="server-config"><java-config>
 
 `<jvm-options>[9|]--add-opens=java.logging/java.util.logging=ALL-UNNAMED</jvm-options>`  
 
+Add/Confirm the Existence of the Following JVM Option to/in the `<config name="server-config"><java-config>` section:
```

Comment:
I'd drop this for the 5.4 notes - all of the options above are the same case and the note about the path is at the start of step 5 already.